### ISSUE TITLE
Alternative method for changing learning rate multiplier?

### ISSUE THREAD

**Issue by Kelvin945** [pytorch/pytorch#14789]
It seems pytorch didn't have learning rate multiplier parameter for each layer. 

Is there a better way to change learning rate multiplier for different layer?

**Comment by soumith** [comment]
please use https://discuss.pytorch.org for questions
